src/app/core/services/youtube.search.spec.ts

```diff
@@ -2,13 +2,20 @@
  * Testing a Service
  * More info: https://angular.io/docs/ts/latest/guide/testing.html
  */
-import { TestBed, inject } from '@angular/core/testing';
-import { HttpClientModule } from '@angular/common/http';
+import { 
+  TestBed, 
+  inject 
+} from '@angular/core/testing';
+import { 
+  HttpClientModule
+} from '@angular/common/http';
 import { PlayerSearchActions } from '@store/player-search';
 import { YoutubeDataApi } from './youtube-data-api';
 import { YoutubeSearch, SearchTypes } from './youtube.search';
 
-describe('Youtube Search Service', () => {
+describe(
```

Comment:
The `describe` function call can be kept on a single line for better readability, unless the line length exceeds the recommended limit.